### PR TITLE
Fix some rubocop warnings

### PR DESCRIPTION
**Pull request by matugm**
== lib/rails_erd/domain/relationship.rb ==
    W: 73:  9: end at 73, 8 is not aligned with unless at 66, 56

    == examples/identity.rb ==
    W: 36:  5: Useless assignment to variable - macro.
    W: 45:  5: Useless assignment to variable - macro.

    == test/test_helper.rb ==
    W: 37: 20: Ambiguous splat operator. Parenthesize the method arguments if it's surely a splat operator, or add a whitespace to the right of the * if it should be a multiplication.

### CODE REVIEW

**Comment by matugm** [comment]
Not sure why Travis failed, test are passing on my machine with Ruby 2.1

**Comment by kerrizor** [comment]
Thanks for the PR! We're seeing intermittent TravisCI failures; I'll restart the job and it usually will pass then..

File: lib/rails_erd/domain/relationship.rb [hunk 1]
```diff
@@ -63,7 +63,8 @@ def association_target(association)
 
       def initialize(domain, associations) # @private :nodoc:
         @domain = domain
-        @reverse_associations, @forward_associations = *unless any_habtm?(associations)
+        @reverse_associations, @forward_associations =
+        unless any_habtm?(associations)
```
Comment (kerrizor) [inline lib/rails_erd/domain/relationship.rb]: I like this better, but it's still a little hard to read. How about a new private function for this unless/else block?  ..additionally, I'm personally in favor of not using `else` with an `unless` whenever possible; could you flip the logic?

File: test/test_helper.rb [hunk 2]
```diff
@@ -34,7 +34,7 @@ def create_table(table, columns = {}, pk = nil)
   def add_column(*args)
     ActiveRecord::Schema.instance_eval do
       suppress_messages do
-        add_column *args
+        add_column(*args)
```
Comment (kerrizor) [inline test/test_helper.rb]: <3

**Comment by kerrizor** [comment]
Thanks again!

**Pull request merged**